src/signals/incident/components/form/ContainerSelect/context.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FunctionComponent } from 'react';
```

Comment:
This is a component, it has the `.tsx` extension. So, (according to our current naming conventions) the filename should be PascalCased.